where-to-travel/src/main/webapp/index.html

```diff
@@ -19,10 +19,10 @@
       <header class="bmd-layout-header">
         <div class="navbar bg-faded">
           <div class="text-center" style="width:330px">
-            <h1 class="navbar-brand text-center text-color font-size-large"> <img src="icons/pin.svg" width="20" height="38"> Where to?</h1>
+            <h1 class="navbar-brand text-center text-color font-size-large"> <img src="icons/pin.svg" width="20" height="38"> Where to? </h1>
```

Comment:
Did you mean to add this space?

Reply:
Yes, I did it to be consistent with the leading space.